### PR TITLE
Remove intermediate wrapper from morph clients

### LINKED ISSUE #625: Reduce amount of wrappers in `pkg/morph/client`
Currently we have hierarchy of clients talking to our contracts:
1. Raw neo-go client.
2. Static client talking to a specific contract (see also nspcc-dev/neo-go#1967)
3. Low-level wrapper in `pkg/morph/client/balance`
4. High-level wrapper in `pkg/morph/client/balance/wrapper`.

Do we need to separate 3 and 4? I suggest removing one of the levels as it feels artificial and unnecessary verbose.
With additional struct it is also easier to forget to add argument on a higher level. With direct invocation compiler ensures completeness for us.

As @cthulhu-rider noticed, we can have `func Method(args MethogArgs)` in contract and use the struct imported from `neofs-contract` repo directly (this will need client support from neo-go though).

I am OK with any variant as long as it reduces the amount of code needed to write and maintain while increasing readability.

### PR DESCRIPTION
**Pull request by fyrchik**
Depends on #1125 .
Close #625 .

So after this it is hard to argue for having a tower of wrappers, as the interface is the same, but we have 2k LOC less.
It is also faster, because there are no intermediate conversions (including array allocations just to convert element types).

### CODE REVIEW

**Review by cthulhu-rider — APPROVED** [review]

File: pkg/innerring/processors/container/process_eacl.go [hunk 1]
```diff
@@ -42,12 +42,11 @@ func (cp *Processor) checkSetEACL(e container.SetEACL) error {
 	binTable := e.Table()
 	tableHash := sha256.Sum256(binTable)
 
+	// verify the identity of the container owner
```
Comment (cthulhu-rider) [inline pkg/innerring/processors/container/process_eacl.go:45]: Previous position was correct, `key.Verify` just check the signature as said above.

Comment (fyrchik) [reply pkg/innerring/processors/container/process_eacl.go]: Fixed

File: pkg/morph/client/netmap/new_epoch.go [hunk 2]
```diff
@@ -6,27 +6,12 @@ import (
...
-func (c *Client) NewEpoch(args NewEpochArgs) error {
+// NewEpoch updates NeoFS epoch number through
+// Netmap contract call.
+func (c *Client) NewEpoch(epoch uint64) error {
 	prm := client.InvokePrm{}
-
 	prm.SetMethod(newEpochMethod)
-	prm.SetArgs(args.number)
-	prm.InvokePrmOptional = args.InvokePrmOptional
+	prm.SetArgs(epoch)
```
Comment (alexvanin) [inline pkg/morph/client/netmap/new_epoch.go:14]: You should either convert it into `int64` or update code in `pkg/morph/client/client.go#384`. 

```diff
	case int64: // TODO: add other numerical types
		result.Type = sc.IntegerType
+	case uint64:
+		result.Type = sc.IntegerType
+		result.Value = int64(v)
```

I didn't check all the code, maybe there is more cases when we pass `uint64` into args in this PR, so maybe change above is more preferred.

**Review by alexvanin — CHANGES_REQUESTED** [review]

File: pkg/morph/client/netmap/new_epoch.go (hunk 2 above)
Comment (fyrchik) [reply pkg/morph/client/netmap/new_epoch.go:14]: The problem is that this is `smartcontract.Parameter` which has a specific logic when marshaling in JSON https://github.com/nspcc-dev/neo-go/blob/master/pkg/smartcontract/parameter.go#L67 . Similar thing happens during unmarshaling on the server side https://github.com/nspcc-dev/neo-go/blob/01d15ff473f9984023e9c1e04be107981acec80e/pkg/rpc/request/txBuilder.go#L65 .

Comment (fyrchik) [reply pkg/morph/client/netmap/new_epoch.go]: nspcc-dev/neo-go#2350

File: pkg/morph/client/neofs/cheque.go [hunk 3]
```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/nspcc-dev/neofs-node/pkg/morph/client"
 )
 
-// ChequePrm groups the arguments of Cheque operation.
+// ChequePrm groups parameters of AlphabetUpdate operation.
```
Comment (carpawell) [inline pkg/morph/client/neofs/cheque.go:9]: misclick?

File: pkg/morph/client/neofs/bind.go [hunk 4]
```diff
@@ -58,11 +49,15 @@ func (x *Client) BindKeys(args BindKeysArgs) error {
 	return nil
 }
 
+// UnbindKeysPrm groups parameters of BindKeys operation.
```
Comment (carpawell) [inline pkg/morph/client/neofs/bind.go:52]:
```suggestion
// UnbindKeysPrm groups parameters of UnbindKeys operation.
```

File: pkg/morph/client/netmap/config.go [hunk 5]
```diff
@@ -82,42 +175,145 @@ func (s *SetConfigPrm) SetValue(value interface{}) {
...
-	rs []stackitem.Item
+// SetKey sets binary key to configuration parameter.
+func (c *ConfigPrm) SetKey(v []byte) {
+	c.key = v
 }
 
-// IterateRecords iterates over all config records and passes them to f.
+// Config performs the test invoke of get config value
+// method of NeoFS Netmap contract.
+func (c *Client) Config(p ConfigPrm, assert func(stackitem.Item) (interface{}, error)) (interface{}, error) {
```
Comment (carpawell) [inline pkg/morph/client/netmap/config.go:282]: do we need it public if we have getters for all (all, yes?) the config values?

File: pkg/morph/client/audit/client.go [hunk 6]
```diff
@@ -25,7 +27,12 @@ const (
...
 )
 
-// New creates, initializes and returns the Client instance.
-func New(c *client.StaticClient) *Client {
-	return &Client{client: c}
+// NewFromMorph returns the wrapper instance from the raw morph client.
+func NewFromMorph(cli *client.Client, contract util.Uint160, fee fixedn.Fixed8, opts ...client.StaticClientOption) (*Client, error) {
+	sc, err := client.NewStatic(cli, contract, fee, opts...)
+	if err != nil {
+		return nil, err
```
Comment (carpawell) [inline pkg/morph/client/audit/client.go:34]:
```suggestion
		return nil, fmt.Errorf("could not create static client of audit contract: %w", err)
```

to sync with the other constructors

**Review by carpawell — COMMENTED** [review]
what rule did you use when deciding if the parameters of the call should be kept in a group or separated?

File: pkg/morph/client/neofs/cheque.go (hunk 3 above)
Comment (fyrchik) [reply pkg/morph/client/neofs/cheque.go:9]: Fixed

File: pkg/morph/client/neofs/bind.go (hunk 4 above)
Comment (fyrchik) [reply pkg/morph/client/neofs/bind.go:52]: Fixde

File: pkg/morph/client/audit/client.go (hunk 6 above)
Comment (fyrchik) [reply pkg/morph/client/audit/client.go:34]: Fixed.

File: pkg/morph/client/netmap/config.go (hunk 5 above)
Comment (fyrchik) [reply pkg/morph/client/netmap/config.go:282]: No, fixed.

**Comment by fyrchik** [comment]
@carpawell 0-1 parameter methods are used without a structure, 3+ parameter methods are used with structure. Did you notice some inconsistencies?

**Review by alexvanin — APPROVED** [review]

**Comment by carpawell** [comment]
> @carpawell 0-1 parameter methods are used without a structure, 3+ parameter methods are used with structure. Did you notice some inconsistencies?

@fyrchik, just remember that we had it grouped (even if there are no parameters at all) to make it easier to maintain client api interface changes. if some of them are not grouped then i do not see it reasonable to group them at all

**Pull request merged**